system_metrics_collector/test/system_metrics_collector/test_periodic_measurement_node.cpp

```diff
@@ -95,10 +95,9 @@ class TestPeriodicMeasurementNode : public ::system_metrics_collector::PeriodicM
     return kTestMetricName;
   }
 
-  const std::string & GetMetricUnit() const override
+  std::string GetMetricUnit() const override
   {
-    static const std::string unit_name{kTestMetricUnit};
-    return unit_name;
+    return kTestMetricUnit;
```

Comment:
This is a good example of the misses increasing. Since test files are included, lines 98 and 100 are missed because there's nothing calling them that is tested.